### DISCUSSION TITLE
Route [school.view] not defined

### DISCUSSION THREAD

**Discussion by mallemagassa** [filamentphp/filament#5069] [Help]
I get an exception when I try to use the table Builder I would like to Apply a url There - here is my code below:

protected function getTableActions(): array
{
    return [ 
        Tables\Actions\ActionGroup::make([
            Tables\Actions\ViewAction::make()->url(fn (School $record): string => route('school.view', ['school' => $record])),
            Tables\Actions\EditAction::make(),
            Tables\Actions\DeleteAction::make(),
        ]),
    ];
}

It shows me this error: Route [school.view] not defined.

Let me know if you need more details.

**Comment by danharrin** [comment]
Send the code you used to register this route

**Reply by mallemagassa** [reply]
No I haven't saved the router, I'm using the default filament resource. Or when we want to create this kind of URL we must define a route somewhere???

**Reply by danharrin** [reply]
So you have a SchoolResource? `->url(fn (School $record): string => SchoolResource::getUrl('view', ['record' => $record]))`

**Reply by mallemagassa** [reply]
Thank you very much it works now
